script.reddit.reader/resources/lib/converthtml.py

```diff
@@ -155,15 +114,14 @@ def get_alt_and_link(source_url, text):
             pass
         else:          #relative to page
             uri='http://'+domain+url_path+'/'+uri  #not tested
-        
+
     return alt,uri
-    
+
 def listitem(label, plot, link='', art_thumb='', domain='', votes=0 ):
-    #builds the listitem object used in the gui
-    #from resources.lib.utils import unescape
-    liz=xbmcgui.ListItem(label=label, 
+
+    liz=xbmcgui.ListItem(label=label,
                          label2="",
-                         iconImage="", 
```

Comment:
please drop both iconImage and thumbnailImage as they are both deprecated. You're already using setArt below.